services/apps/alcs/src/alcs/local-government/local-government.controller.spec.ts

```diff
@@ -44,14 +45,33 @@ describe('LocalGovernmentController', () => {
   });
 
   it('should call through for list', async () => {
-    const mockGovernment = {
+    const mockGovernment = new LocalGovernment({
+      name: 'Government',
+      uuid: 'uuid',
+      preferredRegion: {
+        code: 'code',
+        label: 'label',
+      } as ApplicationRegion,
+      isActive: false,
```

Comment:
```suggestion
      isActive: true,
```

This probably doesn't affect test result, but the mock government should probably be active so that `listActive` would make more sense